backbone_graph:
  graph_file:
    seg_graph_name: seg_ncs_v2.graph
    det_graph_name: lpr_ncs_v2.graph
    horizon_fine_graph_name: horizon_fine_v2.graph
  networkDim: 300
  target_h: 300
  target_w: 300